app/api/jobs/routine/[id]/executions/route.ts

```diff
@@ -54,19 +55,14 @@ export async function GET(
       );
     }
 
-    const [executions, totalExecutions] = await Promise.all([
-      vellumClient.workflowDeployments.listWorkflowDeploymentEventExecutions(
-        deployment.id,
-        { limit: pageSize, offset: offset }
```

Comment:
we want to keep the limit, offset params we had here. we shouldn't need the second endpoint since this first one should return a `count` field